_announcements/index.md

```diff
@@ -7,7 +7,7 @@ show-title: true
 ---
 <ul>
     {% for doc in site.announcements reversed %}
-        <li><a href="{{site.url}}{{ doc.url }}">{{doc.date | date: "[%B %d, %Y]" }} {{ doc.title }}</a></li>

```

Comment:
The actual announcement is probably the most important part of this page. Let's begin each line with the title of the announcement, followed by the date in parenthesis. Ex. (Pool Employment, October 22, 2019)